src/fw-update/fw-update-factory.cpp

```diff
@@ -17,6 +15,13 @@
 
 namespace librealsense
 {
+    // We keep the L500 recovery identification to not confuse it with D400
```

Comment:
I vote to remove this too.
If I recall, the problem was that L515 With oud DFU on USB2 reported D400 recovery.
This mechanism protect flashing the D400 flash to it.
This is a very rare case and on this release we will even say we do not support L515..

Reply:
But there is a chance that someone will brick their L515 in recovery mode.
If you're sure...